LSP-volar.sublime-settings

```diff
@@ -41,6 +41,16 @@
 			}
 		}
 	},
+	"settings": {
+		"volar.codeLens.references": true,
+		"volar.codeLens.pugTools": false,
+		"volar.autoCompleteRefs": true,
+		"volar.codeLens.scriptSetupTools": false,
+		"volar.completion.autoImportComponent": true,
+		"volar.preferredAttrNameCase": "auto-kebab",
+		"volar.preferredTagNameCase": "auto",
+		"volar.takeOverMode.enabled": "auto"
```

Comment:
Makes even more sense since that setting in volar is extension-specific. The server doesn't do anything with it. So it would be fine to just name it differently here.